### ISSUE TITLE
Closing brackets in text are highlighted

### ISSUE THREAD

**Issue by Nothing4You** [clifford-github/sublime-ansible#34]
22.12.0

```yaml
---

- name: patch some xml
  community.general.xml:
    path: "{{ xml_path }}"
    xpath: /foo/bar[@name='baz']/qux
    state: absent
```

<img width="315" alt="image" src="https://user-images.githubusercontent.com/3694534/205471484-f5680875-84f6-49f2-8dcd-01069ac0109a.png">

**Comment by killua99** [comment]
Great, wasn't expected so detailed feedback at all. Gonna look into the 4 issues you created. Would you like to install this syntax directly from master before tagging a new release, so I could have another one looking at it.

**Comment by Nothing4You** [comment]
Sure, I'll just need to figure out how to install it manually in ST4 on macOS.
The version installed from package control seems to be a zip file at `~/Library/Application Support/Sublime Text/Installed Packages/Ansible.sublime-package`.
Would I clone the repo into `~/Library/Application Support/Sublime Text/Packages` still?
The folder structure seems to have changed a bit since the readme was written.

**Comment by killua99** [comment]
You can search by `Preferences: Browse Packages`  inside the Sublime Text menu, I did already some modifications.

**Comment by Nothing4You** [comment]
this also appears to be resolved in f83b56aa0a236264daa291c24f586fdcb2bedee4
